frontend/src/metabase/dashboard/actions/data-fetching-typed.ts

```diff
@@ -1,3 +1,4 @@
+import { createAction as RTKCreateAction } from "@reduxjs/toolkit";
```

Comment:
I'd prefer to keep the original name

Reply:
it collides with createAction from `redux-actions`